runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessMainView/ProcessMainView.styled.ts

```diff
@@ -24,6 +24,20 @@ export const ProcessInternalPage = styled(InternalPage)`
     }
 `;
 
+export const TablLink = styled(Link)<{ isActive?: boolean }>`
+    && {
+        text-decoration: none;
+        cursor: pointer;    
+        color: ${props => props.isActive ? '#FBB040' : '#000000'};
+        border-bottom: ${props => props.isActive ? '0.15rem solid #EA8E05' : 'none'};
+    }
+
```

Comment:
I think it will be more clear if we use colors as theme variables (these ones are color taken from theme, so I think you can change that)